src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let calls = 0;
+
+  return function getSum(firstNum) {
+    calls++;
+
+    return function(secondNum) {
+      if (calls <= 3 || calls % 2 !== 0) {
+        return firstNum + secondNum;
+      } else {
+        return 'Bzzz... Error!';
+      }
```

Comment:
Can you write code in `else` body outside `else`?

```suggestion
      }
      
      return 'Bzzz... Error!';
```